### ISSUE TITLE
Add option to just use current directory

### ISSUE THREAD

**Issue by Mrono** [moll/mina-rsync#2]
This applies to capistrano too

**Comment by jalcine** [comment]
Yeah, this is what I thought it'd do (use a provided, if not current, directory as the files to be deployed). I'm not 100% sure if that's what @Mrono is looking for, but I've forked just to make a PR about this.

**Edited**: Just going to do this by adding a line and **not** using this plug-in since it looks _very_ unlikely that the maintainer would respond.

**Comment by jalcine** [comment]
Would make this the ideal companion to a Jekyll blog :+1: :8ball:
